### DISCUSSION TITLE
Trouble with custom validation using getValues()

### DISCUSSION THREAD

**Discussion by getsetbro** [react-hook-form/react-hook-form#3515] [Q&A] [answered]
I am having trouble with custom validations that are using getValues(). Trying to have input fields have validation only if the Radio button, that they are in, is selected. I wonder if having a <controller/> in another controller is problematic.

https://codesandbox.io/s/clever-goldstine-litny?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
hmmm controller inside a controller. what's the reason?

**Reply by getsetbro** [reply]
So the user can choose to type their own value as the last of the radio button choices.

**Reply by getsetbro** [reply]
I simplified it - no nested controller - still cannot get the conditional validation. https://codesandbox.io/s/cranky-water-bnpxx?file=/src/App.js

**Reply by bluebill1049** [reply]
https://codesandbox.io/s/hidden-architecture-5sp6o?file=/src/App.js

`trigger` 👍
